modules/exoplanets/rendering/renderableorbitdisc.cpp

```diff
@@ -95,7 +95,7 @@ namespace {
         float size;
 
         // [[codegen::verbatim(EccentricityInfo.description)]]
-        float eccentricity;
+        float eccentricity [[codegen::inrange(0.0, 0.99)]];
```

Comment:
`[[codegen::less(1.0)]]`?  Or we should add a bunch of more 9s here to enable as much eccentric orbit as possible